fronts-client/src/components/FrontsEdit/CardFormInline.tsx

```diff
@@ -446,6 +481,9 @@ class FormComponent extends React.Component<Props, FormComponentState> {
 
     const imageDefined = (img: ImageData | undefined) => img && img.src;
 
+    const setSlideshowHasAtLeastTwoImages = (state: boolean) =>
```

Comment:
This approach will work, but we're mantaining this property as state, when it could be derived from the current form state. I can see why it's necessary for this approach – we need to get the slideshow state out of our FieldArray component somehow.

If it's possible (unsure if you've tried this!), I suspect it'd be simpler to pass through necessary slideshow values to our form directly as props from `mapStateToProps`. This can be done with the `valueSelector` function, which we already use (e.g. [here](https://github.com/guardian/facia-tool/blob/2d8462c6f43cef56401f8cbe24811386ce7c33f1/fronts-client/src/components/FrontsEdit/CardFormInline.tsx#L1049)) to pass through form values so our form can react to them.

V. happy to pair.

Reply:
I knew there would be a way with the form state, but I couldn't find it - will give this a go!